.github/workflows/notebook-weekly-test.yml

```diff
@@ -0,0 +1,55 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+name: Test notebooks that submit jobs
+on:
+  schedule:
+    - cron: "0 0 1,15 * *" # At 00:00 on day-of-month 1 and 15.
+  workflow_dispatch:
+  pull_request:
```

Comment:
Leftover?

Reply:
Sorry; I wasn't quite ready to request the next review. The test is failing but I can't tell why yet. Unfortunately it takes a couple of hours to run so debugging is slow 😢

Reply:
Although now I'm thinking maybe testing in an action is not worth it if a single notebook takes that long?

Reply:
What else would we do? Regular reminder for a human to test? What about one CI workflow per notebook that needs it?

Reply:
I accepted defeat too quickly. I've given the job-submitting workflows internal access and they're going through immediately this morning. Queue times are usually shorter in the mornings (my time) so hopefully these two factors are enough.

Let's continue with the cron job and see how reliable it is.